### ISSUE TITLE
Requests fails with SSL error python 3.6

### ISSUE THREAD

**Issue by FredHappyface** [qpython-android/qpython3#61]
The python requests library installs without a problem however when connecting to api.github.com fails with SSL error no ciphers available. Running the same application in pydroid 3 or with python 3.8 on Debian does not result in this error

**Comment by riverfor** [comment]
Thanks for the feedback, it is indeed a bug, I will fix it as soon.

**Comment by riverfor** [comment]
Hello,
Can you use the below as a temporary solution, I will fix it in next update.

import requests
requests.packages.urllib3.util.ssl_.DEFAULT_CIPHERS = "TLS13-CHACHA20-POLY1305-SHA256:TLS13-AES-128-GCM-SHA256:TLS13-AES-256-GCM-SHA384:ECDHE:!COMPLEMENTOFDEFAULT"

**Comment by FredHappyface** [comment]
Thanks very much for that. Yeah can certainly do :)

**Comment by riverfor** [comment]
If it doesn't work please reopen this issue.

**Comment by riverfor** [comment]
It is fixed in QPython 3L (v3.0.0) which will be released in these two days.

**Comment by phedoff** [comment]
Hi, I tried the latest build (playstore), am still getting the ["no ciphers available" (_ssl.c:841)] error (workaround from comment #3 works).

**Comment by tayloredenterprises** [comment]
I tested this with v3.0.0 and it doesnt work without the import requests syntax defined above.

**Comment by phedoff** [comment]
Reopened in #70 
> I tested this with v3.0.0 and it doesnt work without the import requests syntax defined above.

**Comment by lesca** [comment]
Tested with Qpython 3OS v3.0.0 
still get this error.

**Comment by frank699** [comment]
same problem here :/

**Comment by sibo7851** [comment]
Hey just wanted to mention that this issue still exists, the fix above did seem to do the trick though.

**Comment by Runa53454** [comment]
Same here

**Comment by picogeyer** [comment]
Sorry to bump a closed ticket, but I can't seem to get this to work. I tried the latest Qpython available in the play store and still run into the issue. I then tried the latest version from here: https://github.com/qpython-android/qpython/releases/tag/3.0.1
It looks a lot nicer, but I still run into the same issue.
I go to the QPYPI client and upgrade pip with "pip3 install --upgrade pip"
Then I try to install opencv:
pip3 install opencv-python:

WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError(1, '[SSL: NO_CIPHERS_AVAILABLE] no ciphers available (_ssl.c:841)'),)': /simple/opencv-python/
Could not fetch URL https://pypi.org/simple/opencv-python/: There was a problem confirming the ssl certificate: HTTPSConnectionPool(host='pypi.org', port=443): Max retries exceeded with url: /simple/opencv-python/ (Caused by SSLError(SSLError(1, '[SSL: NO_CIPHERS_AVAILABLE] no ciphers available (_ssl.c:841)'),)) - skipping

Is there perhaps a way to apply the workaround from QPYPI client so that I can install open-cv?
